### PR TITLE
updating WebJobs nugets to 3.0.18

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

**Comment by brettsam** [comment]
@mathewc -- pulling official http extension into script.

File: test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj [hunk 1]
```diff
@@ -39,7 +39,7 @@
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.10.0" />
     <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.1.0" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="2.0.3" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.0-preview2-11736" />
```
Comment (mathewc) [inline test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj:42]: There's another reference to this version in the source code in StorageEndToEndTests

**Review by mathewc — DISMISSED** [review]
Just one comment

File: test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj (hunk 1 above)
Comment (brettsam) [reply test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj:42]: Good catch! Got it.

**Review by mhoeger — APPROVED** [review]
✅

**Pull request merged**